src/lib/tls/msg_client_hello.cpp

```diff
@@ -463,6 +463,24 @@ Client_Hello_12::Client_Hello_12(std::unique_ptr<Client_Hello_Internal> data)
          m_data->extensions().add(new Renegotiation_Extension());
          }
       }
+   // RFC 8446 4.2.3
```

Comment:
I believe this reference is not correct. Can't find the quoted paragraph in RFC 8446. Instead its in [RFC 8422 4](https://www.rfc-editor.org/rfc/rfc8422#section-4).

Reply:
Oops, this happens when you copy paste and forget to adept all fields...